src/pages/login/Login.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './Login.scss';
+
+const Login = () => {
+  return (
+    <div className="haimLogin">
+      <div className="wrapLogin">
```

Comment:
```suggestion
    <div className="login">
      <div className="wrapLogin">
```
가장 최상위 부모태그의 className은 컴포넌트의 이름과 동일하게 해주셔야 합니다.
단 카멜케이스로!

Reply:
네! 수정완료 하였습니다!